File: oshi-front/src/apis/request/sale/UpdateSale.Request.Dto.ts
export default interface UpdateSaleRequestDto {
    saleId: number; // @NotNull -> number 타입
    price: number; // BigDecimal을 number로 변환
    description: string; // @NotNull -> string 타입
    state: string; // SaleStatusEnum -> string 타입으로 변환
    file?: File; // MultipartFile -> File, 선택 필드로 변환
  }

File: ohsi-back/src/main/java/com/oshi/ohsi_back/domain/sale/presentation/dto/request/UpdateSaleRequestDto.java
package com.oshi.ohsi_back.domain.sale.presentation.dto.request;

import java.math.BigDecimal;

import org.springframework.web.multipart.MultipartFile;

import com.oshi.ohsi_back.domain.sale.domain.enums.SaleStatusEnum;

import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class UpdateSaleRequestDto {
    @NotNull
    private int saleId;
    @NotNull
    private BigDecimal price;
    @NotNull
    private String description;
    @NotNull
    private SaleStatusEnum state;
    
    private MultipartFile file;

}
